packages/suite/src/components/suite/labeling/MetadataLabeling/withEditable.tsx

```diff
@@ -88,7 +88,7 @@ export const withEditable =
 
         return (
             <>
-                <WrappedComponent {...props}>
+                <WrappedComponent>
```

Comment:
This way, some props get lost (`icon`, `variant`, etc.).

<img width="370" alt="Screenshot 2024-03-28 at 13 21 46" src="https://github.com/trezor/trezor-suite/assets/42465546/6e1d8c6e-7990-40e6-9f23-2ba970cd870f">

Reply:
:scream: Good catch! Thanks! Fixed.